package.json

```diff
@@ -11,58 +11,60 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
+    "start-prod": "ng serve --prod --aot=true --optimization=true",
```

Comment:
I used this to first see that the examples were missing and then to see that they were present when I found out how to refer to them in the AppModule which prevents them being tree shaken out.

Reply:
Since this was meant only for debugging purposes, do we see any use in still keeping it after the debug was finished?

Reply:
you're right I think this can be removed.